code/modules/food&drinks/kitchen machinery/monkeyrecycler.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/monkey_recycler
 	name = "monkey recycler"
-	desc = "A machine used for recycling dead monkeys into monkey cubes. It requires 5 monkeys per cube."
+	desc = "A machine used for recycling dead monkeys into monkey cubes. It currently produces 1 cube for every 5 monkeys inserted." // except it literally never does
```

Comment:
You don't have to change it but so you know: this doesn't need to be set since RefreshParts(), where the description is updated, is called in New() anytime a recyeler is spawned.